diff --git a/crates/db_schema/src/schema.rs b/crates/db_schema/src/schema.rs
index 255c6422..f2ccf5e2 100644
--- a/crates/db_schema/src/schema.rs
+++ b/crates/db_schema/src/schema.rs
@@ -76,13 +76,13 @@ diesel::table! {
         published -> Timestamptz,
     }
 }
 
 diesel::table! {
     use diesel::sql_types::*;
-    use super::sql_types::Ltree;
+    use diesel_ltree::sql_types::Ltree;
 
     comment (id) {
         id -> Int4,
         creator_id -> Int4,
         post_id -> Int4,
         content -> Text,
