--- schema.rs	2023-06-21 22:25:50.252384233 +0100
+++ "schema copy.rs"	2023-06-21 22:26:50.452378651 +0100
@@ -6,10 +6,6 @@
     pub struct ListingTypeEnum;
 
     #[derive(diesel::sql_types::SqlType)]
-    #[diesel(postgres_type(name = "ltree"))]
-    pub struct Ltree;
-
-    #[derive(diesel::sql_types::SqlType)]
     #[diesel(postgres_type(name = "registration_mode_enum"))]
     pub struct RegistrationModeEnum;
 
@@ -78,7 +74,7 @@
 
 diesel::table! {
     use diesel::sql_types::*;
-    use super::sql_types::Ltree;
+    use diesel_ltree::sql_types::Ltree;
 
     comment (id) {
         id -> Int4,
